metaflow/parameters.py

```diff
@@ -391,6 +419,10 @@ def wrapper(cmd):
         cmd.has_flow_params = True
```

Comment:
`cmd.has_flow_params = True` is used to determine where to dynamically inject user defined parameters...

they are to be injected in commands like `run` and sub-commands such as `trigger` (for `argo-workflows`, etc.)
Note: this is just a comment that I will be referencing later...